### DISCUSSION TITLE
Using Templates in file/folder names

### DISCUSSION THREAD

**Discussion by DimmKirr** [hairyhenderson/gomplate#1548] [Q&A]
Hey all, I could be a bad doc reader, but are there any possibilities to render file names? 
`For instance gomplate -d src/` and `src` has directory structure like this  `{{ .Data.Value1 }}/{{ .Data.Value2 }}.txt`. Running against it would be rendered into `test/value.txt` directory structure.

Thank you!

**Comment by hairyhenderson** [comment]
You might be looking for the `--output-map` flag, to use instead of `-d`:

https://docs.gomplate.ca/usage/#output-map
